pkg/client/connector/install.go

```diff
@@ -147,14 +147,33 @@ func (ki *installer) removeManagerAndAgents(c context.Context, agentsOnly bool,
 		ai := ai // pin it
```

Comment:
similar to the traffic_manager, I'd look through this but then look at the final, cleaned up version before leaving comments, since I likely addressed some of the more obvious comments you had in the cleanup.